core/chains/evm/chain.go

```diff
@@ -96,7 +97,7 @@ func newChain(dbchain types.Chain, opts ChainSetOpts) (*chain, error) {
 	if cfg.EthereumDisabled() {
 		headTracker = &headtracker.NullTracker{}
 	} else if opts.GenHeadTracker == nil {
-		headTrackerLogger, err2 := l.InitServiceLevelLogger(logger.HeadTracker, serviceLogLevels[logger.HeadTracker])
+		headTrackerLogger, err2 := l.InitServiceLevelLogger(context.TODO(), logger.HeadTracker, serviceLogLevels[logger.HeadTracker])
```

Comment:
I started wiring this up the stack, but there are tons of uses upstream. Can circle back if if the method signature stays though.